controller/array_action/array_mediator_interface.py

```diff
@@ -221,13 +209,14 @@ def get_snapshot(self, snapshot_name, pool_id=None):
         raise NotImplementedError
 
     @abstractmethod
-    def get_snapshot_by_id(self, snapshot_id):
+    def get_object_by_id(self, object_id, object_type):
         """
-        This function return snapshot info about the snapshot.
+        This function return info about volume or snapshot.
         Args:
-            snapshot_id : id of the snapshot in the storage system
+            object_id   : id of the object in the storage system
+            object_type : volume or snapshot
         Returns:
-           Snapshot
+           Snapshot or Volume
         Raises:
             SnapshotIdBelongsToVolumeError
```

Comment:
what is this still doing here?

Reply:
what the problem?

Reply:
do we still have this error in the code?

Reply:
if not, for each error that you remove in this PR, please do not leave any irrelevant traces of it

Reply:
oh.. you meant the error.. I fixed on the green line..

Reply:
done